internal/controller/streams.go

```diff
@@ -272,7 +271,10 @@ func (sm *streamManager) StreamResources(ctx context.Context, logger *zap.Logger
 
 		resourceVersion, err := resourceManager.DynamicListResources(ctx, resourceManager.logger, apiGroup)
 		if err != nil {
-			resourceManager.logger.Error("Failed to list resource", zap.Error(err))
+			if apierrors.IsForbidden(err) {
+				logger.Warn("Access forbidden for resource", zap.String("resource", resource), zap.String("apiGroup", apiGroup), zap.Error(err))
```

Comment:
Nit. We consistently use camel_case for structured log fields.
```suggestion
				logger.Warn("Access forbidden for resource", zap.String("resource", resource), zap.String("api_group", apiGroup), zap.Error(err))
```